lib/rtcpeerconnection/chrome.js

```diff
@@ -427,16 +464,27 @@ function setDescription(peerConnection, local, description) {
     }
   }
 
-  return promise || peerConnection._peerConnection[setLocalDescription](unwrap(description));
+  return promise || peerConnection._peerConnection[setLocalDescription](unwrap(description)).then(function() {
+    if (!local && isPlanB) {
+      maybeDispatchTrackEvents(peerConnection, mediaStreamTracks);
```

Comment:
Are we dispatching events twice here? Once in setRemoteAnswer below and then here.

Reply:
@syerrapragada 

No, because if `setRemoteAnswer()` is called, `promise` will not be undefined and therefore the second `setRemoteDescription` call will not be made.